menu_of_workflows/tails.com/ci_cd/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# NOTE: Set this to the same version you're running in production.
+dbt-snowflake==1.0.0
+colorama
+idna
+snowflake-connector-python
+pycryptodome
+# NOTE: It's best to keep this fixed to a version for stability, and this
+# flow requires at least 0.10.1. Do update to a later versions as they are released.
+sqlfluff==0.10.1
+sqlfluff-templater-dbt==0.10.1
```

Comment:
Would recommend just using `dbt-snowflake~=1.0.0` and `sqlfluff-templater-dbt==0.10.1`, and adding those to the tox.ini deps

Reply:
Agreed on stripping the other deps. Why add them to the tox file? That's not what gets called to set up the environment.

@NiallRees - can you clarify what you mean?

I wonder if using a `tox.ini` file here is distracting because it's only really being used as a `.sqlfluff` file.